docs/book/v1/intro.md

```diff
@@ -42,6 +40,37 @@ if a user has been authenticated or not, e.g. it can be used to verify the
 authorization level of a user (for this scope, it is consumed by
 [zend-expressive-authorization](https://github.com/zendframework/zend-expressive-authorization)).
 
+## Default User class
+
+We provide a default implementation of `UserInterface` via the class `Zend\Expressive\Authentication\DefaultUser`.
+This class is a basic implementation of `UserInterface`. This class is final and
+it's immutable, that means you cannot change it state at runtime.
+In order to set the identity and the user's role we provided a default factory
+class that generates a `UserInterface` object. This factory is
+`Zend\Expressive\Authentication\UserInterfaceFactory`. This class uses a `generate`
```

Comment:
Rephrase to:

```markdown
We provide a factory class for generating `DefaultUser` instances via `Zend\Expressive\Authentication\UserFactory`.
```

Note: I think the name of the factory should either be `UserFactory` or `DefaultUserFactory`, and not contain the word `Interface`.

Additionally, if we will expect a `generate()` method, we likely need to provide an interface to describe the factory, to ensure all implementations are written the same.

Reply:
I'm rewriting the factory using `__invoke()`, so we don't need to specify another interface.